src/main/java/internetshop/service/impl/UserServiceImpl.java

```diff
@@ -37,4 +37,9 @@ public User update(User user) {
     public boolean delete(Long id) {
         return userDao.delete(id);
     }
+
+    @Override
+    public User findByLogin(String login) {
+        return null;
+    }
```

Comment:
it should be in next task, make commit to delete it from here

Reply:
ok